packages/framework/src/hooks/useTemplateData.ts

```diff
@@ -22,6 +23,7 @@ export interface TemplateDataProps {
 export const useTemplateData = ({
   data,
   name = "data",
+  context = {},
```

Comment:
Make this optional instead of default arg

Reply:
But if we make it optional then we will get context as undefined in case it is not passed in, and I think that will be making some issue while merging with the customScope.
is this right ?